telegram/constants.py

```diff
@@ -38,6 +38,9 @@
 Attributes:
     MAX_MESSAGE_ENTITIES (:obj:`int`): 100 (Beyond this cap telegram will simply ignore further
         formatting styles)
+    ANONYMOUS_ADMIN_ID (:obj:`int`):777000
+    CHANNEL_FORWARDER_SERVICE_ID (:obj:`int`): 1087968824 (Telegram service that forwards messages
```

Comment:
```suggestion
    CHANNEL_FORWARDER_SERVICE_ID (:obj:`int`): ``1087968824`` (Telegram service that forwards messages
```

Also this is the anonymous admin id and 777000 is the service chat ;) I'm not sure about the naming, though. 777000 is also the id of the chat that gives you your login PINs. imo naming ``SERVICE_CHAT_ID`` + the note that this is the one forwarding the messages to discussion groups is more concise.
But I'm not too passionate about it …